src/index.js

```diff
@@ -4,6 +4,6 @@ import { getStyles } from './getStyles';
 import { configSchema, defaultConfig } from './configs';
 import { getDataProps } from './getDataProps';
 
-const factory = createFactory(StarterBlockFactory, getStyles, configSchema);
+const factory = createFactory(StarterBlockFactory, { getStyles, configSchema });
```

Comment:
**Question:** We talked about updating `configSchema` to `getConfigSchema`. Is this the right PR to implement that?

Reply:
Yeah, I think it's fine to do it in this one. Implemented.